pywinauto/mouse.py

```diff
@@ -189,15 +196,21 @@ def _perform_click_input(
 
 else:
     _display = Display()
+
+    # TODO: check this method
+    def _get_cursor_pos():
+        data = _display.screen().root.query_pointer()._data
+        return data["root_x"], data["root_y"]
+
     def _perform_click_input(button='left', coords=(0, 0),
                              button_down=True, button_up=True, double=False,
-                             wheel_dist=0, pressed="", key_down=True, key_up=True):
+                             wheel_dist=0, pressed="", key_down=True, key_up=True,
+                             fast_move=False):
         """Perform a click action using Python-xlib"""
         #Move mouse
         x = int(coords[0])
         y = int(coords[1])
         fake_input(_display, X.MotionNotify, x=x, y=y)
-        _display.sync()
```

Comment:
What is the reason to remove it? And how `fast_move` is used here?

Reply:
This is mistake. It should be like this:
```
        if not fast_move:
            _display.sync()
```
I will correct it

Reply:
fixed c06c2c1dec44259598ef3fe5ffd57faff771e323